thoth/storages/graph/postgres.py

```diff
@@ -2983,6 +2991,10 @@ def _construct_software_environments_query(
         python_version: Optional[str] = None,
         cuda_version: Optional[str] = None,
         image_name: Optional[str] = None,
+        library_name: Optional[str] = None,
+        symbol: Optional[str] = None,
+        python_package: Optional[list] = None,
```

Comment:
can we use `package_name` to be consistent?

Reply:
actually package names if this is a list

Reply:
Yes, and this is actually a `str`, thank you :+1: